### PR TITLE
W-11719881 adding hosting section

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -13,6 +13,17 @@ image::api-led-architecture.png[Diagram of structured API layers for experience,
 Anypoint Platform enables you to build a structured application network that uses reusable APIs to connect applications, data, and devices.
 Anypoint Platform makes it easy to discover, create, and manage APIs in a modular, organized layer. Instead of retrieving random (and possibly unstable) code snippets, you can discover and use APIs that are created using the industry’s best practices.
 
+== Hosting Anypoint Platform
+
+Depending on your security, data privacy, and legal requirements, you can host components of Anypoint Platform in the cloud, in a physical data center, or in both locations. 
+
+The control plane consists of Anypoint Platform components that you use to design, deploy, and manage APIs and Mule applications. The runtime plane consists of Anypoint Platform components where your APIs and Mule applications are deployed and made available to your users.
```
Comment (dukesphere) [inline modules/ROOT/pages/index.adoc]:
```suggestion
The control plane consists of Anypoint Platform components that you use to design, deploy, and manage APIs and Mule applications. The runtime plane is where your APIs and Mule applications are deployed and made available to your users.
```

**Review by dukesphere — COMMENTED** [review]
@IsaacEldridge could you gloss it with something like this?

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/index.adoc]: alternatively
```suggestion
The control plane consists of Anypoint Platform components that you use to design, deploy, and manage APIs and Mule applications. The runtime plane hosts your APIs and Mule application deployments and makes them available to your users.
```

**Review by dukesphere — COMMENTED** [review]
altenative

**Review by hannanelson — APPROVED** [review]

**Pull request merged**